src/Table.luna

```diff
@@ -524,20 +747,131 @@ class Table:
         self.setAt name r
 
     # sortBy :: [(Column, SortOrder, NullPosition)]
+    # Sorts the table by given columns. The sorting order can be ascending or descending and nulls can be at the end or beginning of a sorted column. 
```

Comment:
It might be worth adding that lexicographical order is used. (so it is clear how function behaves when many columns are given)